RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -13,25 +13,16 @@ void print_footer() {
 
 boolean auto_pre_adventure()
 {
-	auto_log_debug("Running auto_pre_adv.ash");
-
 	location place = my_location();
-	if((equipped_item($slot[familiar]) == $item[none]) && (my_familiar() != $familiar[none]) && (auto_my_path() == "Heavy Rains"))
-	{
-		abort("Familiar has no equipment, WTF");
-	}
```

Comment:
1. this abort was happening before we actually had a chance to set the familiar equip (a few lines below).
2. this abort is unnecessary because we already have special handling for heavy rains in `void preAdvUpdateFamiliar(location place)`